### PR TITLE
Fixes email flow

### PR DESCRIPTION
**Pull request by smyzin**
- fixed tiny message
- fixed buttons

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -55,6 +58,11 @@ const commonClasses = (
     boxShadow('shadow-2xl', 'hover:shadow-lg', 'active:shadow-md'),
     width(fullWidth ? 'w-full' : 'w-fit'),
     textAlign(center ? 'text-center' : undefined),
+    fontSize(small ? 'text-sm' : 'text-lg'),
+    lineHeight(small ? 'leading-5' : 'leading-7'),
+    type !== 'tertiary'
+      ? padding(small ? 'py-2' : 'py-4', small ? 'px-4' : 'px-6')
```
Comment (T-Damer) [inline src/components/Button.tsx:64]: I think doing the `small ? {'py-2': true, 'px-4': true} : ...` is faster than checking for `small` two times

File: src/components/Text.tsx [hunk 2]
```diff
@@ -286,14 +286,19 @@ const tinyText = (color: 'base' | 'primary' | 'error', primary?: boolean) =>
...
+  fontPrimary,
   children,
-}: { color?: 'base' | 'primary' | 'error'; primary?: boolean } & ChildrenProp) {
-  return <div className={tinyText(color || 'base', primary)}>{children}</div>
+}: {
+  color?: 'base' | 'primary' | 'error'
+  fontPrimary?: boolean
+} & ChildrenProp) {
+  return (
+    <div className={tinyText(color || 'base', fontPrimary)}>{children}</div>
```
Comment (T-Damer) [inline src/components/Text.tsx:302]: Why don't we do `color = 'base'` in the props?

**Review by T-Damer — APPROVED** [review]
Mostly cool, let's make sure it's clean

File: src/components/Button.tsx (hunk 1 above)
Comment (smyzin) [reply src/components/Button.tsx]: Idk about that. Thanks)

File: src/components/Text.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/Text.tsx]: You are right. Changed it

**Pull request merged**